Resources/public/scss/modules/content-tree/_list.item.scss

```diff
@@ -91,6 +90,23 @@ $list-item-height: calculateRem(20px);
         .ez-icon {
             fill: $ez-color-base-dark;
         }
+
+        &--loading-limit-reached {
+            color: $ez-secondary-ground;
+            font-style: italic;
+
+            &.btn[disabled] {
+                opacity: 1;
```

Comment:
Why we want the disabled button to have full opacity? The lower opacity was added to show that you cannot click on button.

Reply:
Because in wireframes we have specific color and with opacity 0.3 color is different.

On the second thought, maybe better approach would be to show "Load limit reached" information as separate div, not "Show more" button. We would avoid this weird styling + the logic in `renderLoadMoreBtn` would be simpler. What do you think?

Reply:
Seems like a good idea.